src/openrct2/world/Footpath.cpp

```diff
@@ -1912,45 +1909,44 @@ void footpath_update_queue_entrance_banner(int32_t x, int32_t y, TileElement* ti
  *
  *  rct2: 0x006A6B7F
  */
-static void footpath_remove_edges_towards_here(
-    int32_t x, int32_t y, int32_t z, int32_t direction, TileElement* tileElement, bool isQueue)
+static void footpath_remove_edges_towards_here(const CoordsXYZD& footpathPos, TileElement* tileElement, bool isQueue)
```

Comment:
Don't combine direction into the coordinate. Again this represents a direction not the footpathPos.